asset/snippets/test/samples.test.js

```diff
@@ -259,4 +288,44 @@ describe('quickstart sample tests', () => {
     await metadataTable.delete();
     await resultsTable.delete();
   });
+
+  it('should get effective iam policies successfully', async () => {
+    const assetName = `//storage.googleapis.com/${bucketName}`;
+    const stdout = execSync(`node getBatchEffectiveIamPolicies ${assetName}`);
+    assert.include(stdout, assetName);
+  });
+
+  it('should create saved query successfully', async () => {
+    const description = 'description';
+    await createSavedQuery(queryId, description);
+    assert.include(console.log.firstCall.args, savedQueryFullName);
+  });
```

Comment:
generally speaking, I think testing the logs is a bit fragile since this response format could change. I'd test the actual outcome you want, vs. logs, unless you want to specifically test that logs are created (but I don't think this is the same as testing the functionality of the sample). Also NB, mostly just an idea for making tests less fragile.